### PR TITLE
Improve Style/FrozenStringLiteralComment message

### PR DESCRIPTION
**Pull request by burke**
We've had a bunch of developers confused by the previous message; I think this one is more clear.

Specifically, unless the user knows about this specific cop being enabled, the annotation "Missing frozen string literal comment" on a line like `require 'test_helper'` doesn't really educate them on how to proceed.

-----------------

Before submitting the PR make sure the following are checked:

* [x] Wrote [good commit messages][1].
* [x] Commit message starts with `[Fix #issue-number]` (if the related issue exists). *(N/A)*
* [x] Used the same coding conventions as the rest of the project.
* [x] Feature branch is up-to-date with `master` (if not - rebase it).
* [x] Squashed related commits together.
* [x] Added tests. *(N/A)*
* [x] Added an entry to the [Changelog](../blob/master/CHANGELOG.md) if the new code introduces user-observable changes. See [changelog entry format](../blob/master/CONTRIBUTING.md#changelog-entry-format).  *(N/A)*
* [x] All tests are passing.
* [x] The new code doesn't generate RuboCop offenses.
* [x] The PR relates to *only* one subject with a clear title
  and description in grammatically correct, complete sentences.
* [x] Updated cop documentation with `rake generate_cops_documentation` (required only when you've added a new cop or changed the configuration/documentation of an existing cop). *(N/A)*

[1]: http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html

### CODE REVIEW

**Comment by burke** [comment]
Noticed CI was red because of too-long lines. Fixed, sorry!

File: lib/rubocop/cop/style/frozen_string_literal_comment.rb [hunk 1]
```diff
@@ -12,7 +12,7 @@ class FrozenStringLiteralComment < Cop
         include ConfigurableEnforcedStyle
         include FrozenStringLiteral
 
-        MSG = 'Missing frozen string literal comment.'.freeze
+        MSG = 'File must begin with "# frozen_string_literal: true".'.freeze
```
Comment (bbatsov) [inline lib/rubocop/cop/style/frozen_string_literal_comment.rb]: I think that's a bit confusing as well. The file should really begin with this - keep in mind there things like shebangs and other magic comments. Also - code snippets in messages should be in backticks, not double quotes.

Comment (burke) [inline lib/rubocop/cop/style/frozen_string_literal_comment.rb]: Ok. How about something like:

```ruby
MSG = 'Comment must be present near top of file: `# frozen_string_literal: true`.'
```

It's a difficult thing to phrase clearly without getting too wordy :/

Comment (bbatsov) [inline lib/rubocop/cop/style/frozen_string_literal_comment.rb]: How about `Missing magic comment `...`.`? That seems clear enough to me.

Comment (burke) [inline lib/rubocop/cop/style/frozen_string_literal_comment.rb]: Works for me. Updated.

**Comment by bbatsov** [comment]
Thanks! 👍
